### DISCUSSION TITLE
메인 페이지, 테스트 페이지, 결과페이지 디자인 필요

### DISCUSSION THREAD

**Discussion by ohilv3r** [DEV-Yangkong/MoodCraft#4] [Announcements]
https://www.figma.com/file/O4Q4WcmiaT7qvmcLvUuFnU/%EC%A3%BC%EB%A5%98-%EC%8B%AC%EB%A6%AC%ED%85%8C%EC%8A%A4%ED%8A%B8?type=design&node-id=0%3A1&mode=design&t=9Jdt6d31uKuXxnb9-1

**Comment by ohilv3r** [comment]
@sds7629 진우님 아까 회의해서 변동사항 피그마에 적어놨어요 각자 어떤걸 구현할수있는지 오늘 생각해보고 공부하기로했구요. 내일 진우님 오면 결과 산출 방법에 대해 토의하려고합니다.

**Reply by sds7629** [reply]
넵 감사합니다!
